pkg/gadgets/top/tcp/types/types.go

```diff
@@ -44,18 +44,54 @@ func ParseFilterByFamily(family string) (int32, error) {
 type Stats struct {
 	eventtypes.CommonData
 
-	Saddr     string `json:"saddr,omitempty" column:"saddr"`
-	Daddr     string `json:"daddr,omitempty" column:"daddr"`
-	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns"`
-	Pid       int32  `json:"pid,omitempty" column:"pid"`
-	Comm      string `json:"comm,omitempty" column:"comm"`
-	Sport     uint16 `json:"sport,omitempty" column:"sport"`
-	Dport     uint16 `json:"dport,omitempty" column:"dport"`
-	Family    uint16 `json:"family,omitempty" column:"family"`
-	Sent      uint64 `json:"sent,omitempty" column:"sent"`
-	Received  uint64 `json:"received,omitempty" column:"received"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,template:ns,hide"`
+	Pid       int32  `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm      string `json:"comm,omitempty" column:"comm,template:comm"`
+	Family    uint16 `json:"family,omitempty" column:"family,maxWidth:6"`
```

Comment:
In the previous `docs/guides/top/tcp.md` the column was called `IPv`, but the previous implementation was `IP` without the v.
I somehow missed that and defaulted to `family`

Reply:
We use the following in tcp / tcpconnect: 

https://github.com/inspektor-gadget/inspektor-gadget/blob/ad6601f18617986222ad80e419963e542b671846/pkg/gadgets/trace/tcp/types/types.go#L28

https://github.com/inspektor-gadget/inspektor-gadget/blob/ad6601f18617986222ad80e419963e542b671846/pkg/gadgets/trace/tcpconnect/types/types.go#L28

I think we should use the same. Otherwise we have to discuss and align all gadgets.

Reply:
Changed columnname to `ip` with same column settings as in `trace tcp` and `trace tcpconnect`.